### ISSUE TITLE
Adding authorship for an add-on should ask the new author for confirmation

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons#340]
### Describe the problem and steps to reproduce it:

1. A developer has an add-on.
2. They add another user U as an author.

### What happened?

U is automatically added as an author.

### What did you expect to happen?

U should confirm that they want to be an author of the add-on.

### Anything else we should know?
Currently, a notification email will be sent to U, but there is nothing U can do if they don't want to be associated with the add-on. I see various scenarios where this can be deceiving for users, ie an add-on coming from a popular developer account.

Adding @jvillalobos to confirm.

**Comment by jvillalobos** [comment]
Yes, this is a problem. Since there's a notification at least it's hard to fake ownership, but it's still somewhat effective if a developer is slow to react. Not an urgent issue IMO.

**Comment by wagnerand** [comment]
Agreed, not urgent.
